templates/tips/index.html

```diff
@@ -0,0 +1,84 @@
+                  <span class="navbar-toggler-icon"></span>
+                </button>
+
+                <div class="collapse navbar-collapse" id="navbarNavDarkDropdown">
+                  <ul class="navbar-nav">
+                    <li class="nav-item">
+                      <a class="nav-link" type="button" href="add_tip/" role="button">Add new tip</a>
+                    </li>
+                    <li class="nav-item dropdown">
+                      <a class="nav-link dropdown-toggle" href="#" id="navbarDarkDropdownMenuLink" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+                        Categories
+                      </a>
+                      <ul class="dropdown-menu dropdown-menu-dark" aria-labelledby="navbarDarkDropdownMenuLink">
+                        <li><a class="dropdown-item" type="button" href="category/Rent" role="button">Rent</a></li>
+                        <li><a class="dropdown-item" type="button" href="category/Mortage" role="button">Mortgage</a></li>
+                        <li><a class="dropdown-item" type="button" href="category/Bills" role="button">Bills</a></li>
+                        <li><a class="dropdown-item" type="button" href="category/Transportaion" role="button">Transportation</a></li>
+                        <li><a class="dropdown-item" type="button" href="category/Clothing" role="button">Clothing</a></li>
+                        <li><a class="dropdown-item" type="button" href="category/Healthcare" role="button">Healthcare</a></li>
+                        <li><a class="dropdown-item" type="button" href="category/Food" role="button">Food</a></li>
+                        <li><a class="dropdown-item" type="button" href="category/Kids" role="button">Kids</a></li>
+                        <li><a class="dropdown-item" type="button" href="category/Culture" role="button">Culture</a></li>
+                        <li><a class="dropdown-item" type="button" href="category/vacations" role="button">Vacations</a></li>
+                        <li><a class="dropdown-item" type="button" href="category/Other" role="button">Other</a></li>
+                      </ul>
+                    </li>
+                  </ul>
+                </div>
+              </div>
+            </nav>
+          {% for tip in tips %}
+            <article class="post vt-post">
+                <div class="row">
+                    <div class="col-xs-12 col-sm-5 col-md-5 col-lg-4">
+                        <div class="post-type post-img">
+                            <a href="#"><img src="https://bootdey.com/img/Content/avatar/avatar1.png" class="img-responsive" alt="image post"></a>
+                        </div>
+                        <div class="author-info author-info-2">
+                            <ul class="list-inline">
+                                <li>
+                                    <div class="info">
+                                        <p>Posted on:</p>
```

Comment:
Same here, I think Last modified date would fit better 👍

Reply:
Waiting for the teamates to review

Reply:
I don't think it is needed